src/components/LockButton.jsx

```diff
@@ -13,6 +13,7 @@ const LockButton = ({onClick, readOnly}) => {
       label={`Switch to ${ readOnly ? 'edit' : 'read-only' } mode`}
       onTouchTap={onClick}
       tooltip={toolTipText}
+      style={{zIndex: 5}}
```

Comment:
@Hopding In lieu of our discussion I invite you to reconsider requesting changes.